admin/admin_test.go

```diff
@@ -1991,3 +1991,326 @@ func TestGetU2FToken(t *testing.T) {
+	}
+
+	lastPage := LogListV2Metadata{}
+	if lastPage.GetNextOffset() != nil {
+		t.Errorf("Expected nil option to represent no more available logs, got a non-nil option")
+	}
+}
+
+// TestLogListV2Metadata ensures that timestamps are properly parsed from log data.
+func TestParseLogV1Timestamp(t *testing.T) {
+	validLog := map[string]interface{}{
+		"timestamp": 1346172820,
+	}
+	timestamp, err := parseLogV1Timestamp(validLog)
+	if err != nil {
+		t.Errorf("Failed to parse log timestamp: %v", err)
+	}
+	if expectedTs := time.Unix(1346172820, 0); !timestamp.Equal(expectedTs) {
+		t.Errorf("Parsed incorrect value for log timestamp, expected %v but got: %v", expectedTs, timestamp)
+	}
+}
+
+// getAuthLogsResponse is an example response from the Duo API documentation example: https://duo.com/docs/adminapi#authentication-logs
+const getAuthLogsResponse = `{
+    "response": {
+        "authlogs": [
+            {
+                "access_device": {
+                    "browser": "Chrome",
+                    "browser_version": "67.0.3396.99",
+                    "flash_version": "uninstalled",
+                    "hostname": "null",
+                    "ip": "169.232.89.219",
+                    "java_version": "uninstalled",
+                    "location": {
+                        "city": "Ann Arbor",
+                        "country": "United States",
+                        "state": "Michigan"
+                    },
+                    "os": "Mac OS X",
+                    "os_version": "10.14.1"
+                },
+                "application": {
+                    "key": "DIY231J8BR23QK4UKBY8",
+                    "name": "Microsoft Azure Active Directory"
+                },
+                "auth_device": {
+                    "ip": "192.168.225.254",
+                    "location": {
+                        "city": "Ann Arbor",
+                        "country": "United States",
+                        "state": "Michigan"
+                    },
+                    "name": "My iPhone X (734-555-2342)"
+                },
+                "event_type": "authentication",
+                "factor": "duo_push",
+                "reason": "user_approved",
+                "result": "success",
+                "timestamp": 1532951962,
+                "trusted_endpoint_status": "not trusted",
+                "txid": "340a23e3-23f3-23c1-87dc-1491a23dfdbb",
+                "user": {
+                    "key": "DU3KC77WJ06Y5HIV7XKQ",
+                    "name": "narroway@example.com"
+                }
+            }
+        ],
+        "metadata": {
+            "next_offset": [
+                "1532951895000",
+                "af0ba235-0b33-23c8-bc23-a31aa0231de8"
+            ],
+            "total_objects": 1
+        }
+    },
+    "stat": "OK"
+}`
+
+// TestGetAuthLogs ensures proper functionality of the client.GetAuthLogs method.
+func TestGetAuthLogs(t *testing.T) {
+	var last_request *http.Request
```

Comment:
FWIW - I noticed that this is something that appears more than once already in the repo so I really appreciate you trying to match the existing style! I'll see about putting up a PR which addresses this across the board so that `go-lint` doesn't complain 😄

Reply:
Yep, was just following the style of previous test cases.